sdk/va-report-components/documentation/docs/api/ObjectHandle.md

```diff
@@ -11,6 +11,16 @@ When a report element is assigned new attribute values or removed from the DOM,
 any `ObjectHandles` obtained from that element are invalidated and should be
 discarded.
 
+## Properties
+
+### readyState: string
+Describes the ready state of the report object. When this value changes, a `readyStateChanged` event is fired on the ObjectHandle. 
```

Comment:
It seems a little odd to say "Describes" here.  Maybe just `The ready state of the report object.'?